### PR TITLE
feat(WEB):add landing page files

### PR DESCRIPTION
**Pull request by RiteshAdwani**
Contains index.html and styles.css file of the site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+                            <i class="fa-solid fa-angle-down"></i>
+                        </span>
+                    </p>
+                </div> 
+            </div>
+        </section>
+        <!-- Categories section ends -->
+
+
+      <div class="display-flex">
```
Comment (manas-vyas-30) [inline index.html:164]: Always avoid class name which are same as css property.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,414 @@
+*{
+    box-sizing:border-box;
+}
+
+a{
```
Comment (manas-vyas-30) [inline styles.css:5]: Always avoid adding css directly to `html` tags

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+    flex-direction: column;
+    align-items: center;
+    cursor: pointer;
+}
+
+.cart-icon{
+    margin: 0.5% ;
+}
+
+.deals, .other-deals{
```
Comment (manas-vyas-30) [inline styles.css:68]: you could have reused `deals` class. Always name the class more generic if you think that they will be useful in the future.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+
+.flipkart-logo{
+    margin-right:0;
+}
+
+.flipkart-logo-img{
+    display: block;
+}
+
+input{
```
Comment (manas-vyas-30) [inline styles.css:162]: Always avoid adding css directly to html tags

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+    }
+}
+
+@media (max-width:1500px){
+    img.adv-img{
+        width: 160px;
+    }
+}
+
+@media (max-width:1500px){
```
Comment (manas-vyas-30) [inline styles.css:356]: Why are you changing same css property for same screen size for different times?

**Review by manas-vyas-30 — APPROVED** [review]
@RiteshAdwani Good Job. Responsive design could have been better. Sharing this comments for your reference. Please go through them.

File: index.html (hunk 1 above)
Comment (RiteshAdwani) [reply index.html:164]: Comment addressed.

File: styles.css (hunk 2 above)
Comment (RiteshAdwani) [reply styles.css:5]: Will take care of it from now onwards.

File: styles.css (hunk 3 above)
Comment (RiteshAdwani) [reply styles.css:68]: Previously had the same class name, but had to change some properties later on to make it responsive and hence had to give different name.

File: styles.css (hunk 5 above)
Comment (RiteshAdwani) [reply styles.css:356]: Maybe I forgot to change the screen size at that moment, but will pay more attention to avoid such mistakes.

**Pull request merged**